src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -958,7 +958,7 @@ protected void setPowThreads(int powThreads) {
         String PREVIOUS_EPOCHS_SPENT_ADDRESSES_TXT =
                 "/previousEpochsSpentAddresses1.txt /previousEpochsSpentAddresses2.txt " +
                         "/previousEpochsSpentAddresses3.txt";
-        long GLOBAL_SNAPSHOT_TIME = 1554901200;
+        long GLOBAL_SNAPSHOT_TIME = 1554904800;
```

Comment:
Whats the reason for this change?

Reply:
The `dev` branch was not in sync for some reason...
This is a change from the release of 1.7.0